.github/workflows/pickle.yml

```diff
@@ -87,8 +88,13 @@ jobs:
       - name: Check for existence of pickled files
         id: pickle_check
         uses: ./.github/actions/check_for_pickled
+      - name: Check for STC installation
+        id: stc_check
+        uses: ./.github/actions/check_for_stc
+        with:
+          not_editable: 'false'
```

Comment:
This workflow can be editable or non-editable depending on the contents of `${{ inputs.editable_string }}`. You will need to check this value. If you specify the type of the argument (boolean) in the `check_for_stc` action then you should be able to pass something like `${{ inputs.editable_string == '-e' }}` here

Reply:
The fix for this should be done in this file

Reply:
>  If you specify the type of the argument

Ah unfortunately this only seems possible with a workflow

Reply:
This may also be useful: https://docs.github.com/en/actions/learn-github-actions/expressions#example